contrib/goattracker/patches/fix_makefile.patch

```diff
@@ -0,0 +1,45 @@
+--- a/src/makefile.common	2021-05-15 14:59:32.000000000 +0200
++++ b/src/makefile.common	2024-03-14 11:31:54.572719809 +0100
+@@ -2,8 +2,7 @@
+ 
+ CC?=gcc
+ CXX?=g++
+-CFLAGS+=-Ibme -Iasm -O3
+-CXXFLAGS+=$(CFLAGS) -fpermissive
++CFLAGS+=-Ibme -Iasm
+ 
+ EXE=	$(PREFIX)goattrk2$(SUFFIX) \
+ 	$(PREFIX)gt2reloc$(SUFFIX) \
+@@ -23,7 +22,7 @@
+ asm/asmtab.o asm/chnkpool.o asm/expr.o asm/lexyy.o asm/log.o asm/membuf.o asm/membufio.o asm/namedbuf.o asm/parse.o \
+ asm/pc.o asm/vec.o \
+ bme/bme_gfx.o bme/bme_snd.o bme/bme_win.o bme/bme_mou.o bme/bme_kbd.o bme/bme_io.o bme/bme_end.o bme/bme.o
+-	$(CXX) -o $@ $^ $(LIBS)
++	$(CXX) $(LDFLAGS) -o $@ $^ $(LIBS)

```

Comment:
ah, these ones where you added ldflags also need c/xxflags in there too depending on which is used